### PR TITLE
mcs: support admin/cache http interface in scheduling server

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Ref https://github.com/tikv/pd/issues/5839

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/mcs/scheduling/server/apis/v1/api.go [hunk 1]
```diff
@@ -121,6 +121,9 @@ func NewService(srv *scheserver.Service) *Service {
 func (s *Service) RegisterAdminRouter() {
 	router := s.root.Group("admin")
 	router.PUT("/log", changeLogLevel)
+	cacheRouter := router.Group("cache/regions")
```
Comment (rleungx) [inline pkg/mcs/scheduling/server/apis/v1/api.go:124]: How about only cache or just not create a new group?

Comment (lhy1024) [reply pkg/mcs/scheduling/server/apis/v1/api.go:124]: Do you mean `cacheRouter := s.root.Group("admin/cache/regions")`

Comment (rleungx) [reply pkg/mcs/scheduling/server/apis/v1/api.go:124]: cacheRouter := s.root.Group("cache") or just use router.DELETE("cache/regions", xxx)

File: tests/integrations/mcs/scheduling/api_test.go [hunk 2]
```diff
@@ -218,3 +220,57 @@ func (suite *apiTestSuite) TestAPIForward() {
 		testutil.WithHeader(re, apiutil.ForwardToMicroServiceHeader, "true"))
 	re.NoError(err)
 }
+
+func TestAdminRegionCache(t *testing.T) {
```
Comment (rleungx) [inline tests/integrations/mcs/scheduling/api_test.go:224]: How about also checking if both caches can be cleaned in microservice mode?

Comment (lhy1024) [reply tests/integrations/mcs/scheduling/api_test.go:224]: TestAdminRegionCacheForward tests it by "/pd/api/v1/admin/cache/region"

Comment (rleungx) [reply tests/integrations/mcs/scheduling/api_test.go:224]: What I mean is if we only call the API server's interface, both caches can be cleaned in microservice mode.

**Review by rleungx — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**